fastn-core/src/http.rs

```diff
@@ -513,7 +514,11 @@ pub(crate) async fn http_get_with_cookie(
             res.text().await
         );
         tracing::error!(url = url, msg = message);
-        NOT_FOUND_CACHE.write().insert(url.to_string());
+
+        if use_cache {
+            NOT_FOUND_CACHE.write().insert(url.to_string());
+        }
```

Comment:
```suggestion
       NOT_FOUND_CACHE.write().insert(url.to_string());
```
We have already checked this condition above.

Reply:
shouldn't we not insert entries from http processors if the `use_cache` is `false`? This will reduce memory usage

Reply:
I have misread the code. I thought it was inside `if use_cache && NOT_FOUND_CACHE.read().contains(url) {` block. You are right.